packer/install-dependencies.sh

```diff
@@ -33,6 +33,88 @@ installContainerd
 
 installImg
 
+DASHBOARD_VERSIONS="1.8.3 1.6.3"
+for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}; do
+    pullImg "k8s.gcr.io/kubernetes-dashboard-amd64:v${DASHBOARD_VERSION}"
+done
+
+EXECHEALTHZ_VERSIONS="1.2"
+for EXECHEALTHZ_VERSION in ${EXECHEALTHZ_VERSIONS}; do
+    pullImg "k8s.gcr.io/exechealthz-amd64:${EXECHEALTHZ_VERSION}"
+done
+
+ADDON_RESIZER_VERSIONS="1.8.1 1.7"
+for ADDON_RESIZER_VERSION in ${ADDON_RESIZER_VERSIONS}; do
+    pullImg "k8s.gcr.io/addon-resizer:${ADDON_RESIZER_VERSION}"
+done
+
+HEAPSTER_VERSIONS="1.5.3 1.5.1"
+for HEAPSTER_VERSION in ${HEAPSTER_VERSIONS}; do
+    pullImg "k8s.gcr.io/heapster-amd64:v${HEAPSTER_VERSION}"
+done
+
+METRICS_SERVER_VERSIONS="0.2.1"
+for METRICS_SERVER_VERSION in ${METRICS_SERVER_VERSIONS}; do
+    pullImg "k8s.gcr.io/metrics-server-amd64:v${METRICS_SERVER_VERSION}"
+done
+
+KUBE_DNS_VERSIONS="1.14.10 1.14.8 1.14.5"
+for KUBE_DNS_VERSION in ${KUBE_DNS_VERSIONS}; do
+    pullImg "k8s.gcr.io/k8s-dns-kube-dns-amd64:${KUBE_DNS_VERSION}"
+done
+
+KUBE_ADDON_MANAGER_VERSIONS="8.6"
+for KUBE_ADDON_MANAGER_VERSION in ${KUBE_ADDON_MANAGER_VERSIONS}; do
+    pullImg "k8s.gcr.io/kube-addon-manager-amd64:v${KUBE_ADDON_MANAGER_VERSION}"
+done
+
+KUBE_DNS_MASQ_VERSIONS="1.14.10 1.14.8 1.14.5"
+for KUBE_DNS_MASQ_VERSION in ${KUBE_DNS_MASQ_VERSIONS}; do
+    pullImg "k8s.gcr.io/k8s-dns-dnsmasq-nanny-amd64:${KUBE_DNS_MASQ_VERSION}"
+done
+
+PAUSE_VERSIONS="3.1"
+for PAUSE_VERSION in ${PAUSE_VERSIONS}; do
+    pullImg "k8s.gcr.io/pause-amd64:${PAUSE_VERSION}"
+done
+
+TILLER_VERSIONS="2.8.1"
+for TILLER_VERSION in ${TILLER_VERSIONS}; do
+    pullImg "gcr.io/kubernetes-helm/tiller:v${TILLER_VERSION}"
+done
+
+RESCHEDULER_VERSIONS="0.4.0 0.3.1"
+for RESCHEDULER_VERSION in ${RESCHEDULER_VERSIONS}; do
+    pullImg "k8s.gcr.io/rescheduler:v${RESCHEDULER_VERSION}"
+done
+
+VIRTUAL_KUBELET_VERSIONS="latest"
```

Comment:
are we not pinning to a specific version on purpose here?

Reply:
cf. https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/k8s_versions.go#L24